addons/project/models/project.py

```diff
@@ -1374,6 +1374,11 @@ def SELF_READABLE_FIELDS(self):
     def SELF_WRITABLE_FIELDS(self):
         return PROJECT_TASK_WRITABLE_FIELDS
 
+    @api.depends('project_id.color')
+    def _compute_project_color(self):
+        for task in self:
+            task.project_color = task.color if task.color > 0 else task.project_id.color
```

Comment:
But this field had been suppress, and I accidentally re-introduced it. Idea was that having a related (or computed for that matter) field wasn't worth it.
Now, it might have been a little rough.
BUT, we use worksheet_color for the same purpose. So the compromise I found with aju was merging worksheet_color and project_color in one field display_color with the same logic. :pray: